src/strings/en-US/PowerFxResources.resw

```diff
@@ -1922,8 +1922,12 @@
   <data name="AboutTime_millisecond" xml:space="preserve">
     <value>The millisecond component.</value>
   </data>
+  <data name="AboutDateTime" xml:space="preserve">
+    <value>Creates a new date/time value, given its components.</value>
```

Comment:
>date/time [](http://example.com/codeflow?start=25&length=9)

Creates a value that represents an instant in time, expressed as a date and time of the day.